engine/src/main/java/org/terasology/engine/physics/engine/PhysicsSystem.java

```diff
@@ -66,8 +65,6 @@
     @In
     private NetworkSystem networkSystem;
     @In
-    private EntityManager entityManager;
```

Comment:
This was seemingly not used.

Reply:
this whole class is dropped in here: https://github.com/MovingBlocks/Terasology/pull/4853

Reply:
Even better =) I'll drop this change from this PR then to avoid any conflicts :+1: